src/ufoLib2/objects/font.py

```diff
@@ -202,56 +204,72 @@ def read(cls, reader, lazy=True):
             self._reader = reader
         return self
 
-    def __contains__(self, name):
-        return name in self.layers.defaultLayer
-
-    def __delitem__(self, name):
-        del self.layers.defaultLayer[name]
-
-    def __getitem__(self, name):
-        return self.layers.defaultLayer[name]
-
-    def __setitem__(self, name, glyph):
-        self.layers.defaultLayer[name] = glyph
-
-    def __iter__(self):
-        return iter(self.layers.defaultLayer)
-
-    def __len__(self):
-        return len(self.layers.defaultLayer)
-
-    def get(self, name, default=None):
+    def __contains__(self, name: str) -> bool:
+        if self.layers.defaultLayer is not None:
+            return name in self.layers.defaultLayer
+        return False
```

Comment:
Why is this one not raising? To me it would make sense that the font as a whole refuses to act as a bag of glyphs when the default layer is absent, this method shouldn't be particular?